### ISSUE TITLE
Gerbera Debian/Ubuntu repo does not contain packages for armhf

### ISSUE THREAD

**Issue by emk2203** [gerbera/gerbera#1467]
After learning that there is a repository with the latest versions under https://gerbera.jfrog.io, I tried to replace the Ubuntu ppa with it, but failed due to lack of a armhf version available.

Could you supply for this architecture as well via the repo?

**Comment by whyman** [comment]
Hi @emk2203.

This is a nice idea, realistically I dont have time to look at this soon, but patches are definitely welcome if you want to try it.

The `docker` builds do include ARMv7/ARM64 arches.

**Comment by hmoffatt** [comment]
By the looks of things you should be able to run the build-deb.sh script on your own armhf machine to build the package yourself. The only problem I can see is that it installs cmake from a third party repository which make not have armhf binaries either. I'd hack it and see if the cmake you have is recent enough.

It also messes with your installed gcc/g++ versions which is weird - would be better to fix the code to compile with gcc 10 rather than stick to an old version.

**Comment by emk2203** [comment]
@hmoffatt: Yes, I did a manual install before I used the ppa. But I want to have an automated update path via `apt upgrade`.

@whyman: If the docker builds already contain the ARMv7 builds, which should be usable for a armhf distribution, can't this just be offered as a deb package? I admit I have no idea about the behind-the-scenes stuff necessary. What exactly is needed to make this work if the ARMv7 version already exists?
